cmake/projects/chaiscript/hunter.cmake

```diff
@@ -0,0 +1,32 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+  PACKAGE_NAME
+  chaiscript
+  VERSION
+  "6.0.0"
+  URL
+  "https://github.com/hunter-packages/ChaiScript/archive/v6.0.0-hunter.zip"
+  SHA1
+  fb6bf41f03cdebf377b6ae4cae6490c65e584486
+  )
+
+hunter_cmake_args(
+    chaiscript
+    CMAKE_ARGS
+        BUILD_TESTING=OFF
+#        UNIT_TEST_LIGHT=TRUE
```

Comment:
To remove.

Reply:
What specifically here?  The # commented lines?

Reply:
Yes, if you want to mention some important options it's better to use documentation. For example see [pkg.boost](https://docs.hunter.sh/en/latest/packages/pkg/Boost.html#cmake-options).